bigquery/tests/unit/test_job.py

```diff
@@ -1640,6 +1640,39 @@ def test_cancelled(self):
 
         self.assertTrue(job.cancelled())
 
+    def test_query_plan(self):
+        plan_entries = [{
+            'name': 'NAME',
+            'id': 1234,
+            'waitRatioAvg': 2.71828,
+            'waitRatioMax': 3.14159,
+            'readRatioAvg': 1.41421,
+            'readRatioMax': 1.73205,
+            'computeRatioAvg': 0.69315,
+            'computeRatioMax': 1.09861,
+            'writeRatioAvg': 3.32193,
+            'writeRatioMax': 2.30258,
+            'recordsRead': 100,
+            'recordsWritten': 1,
```

Comment:
Ditto. This is a string in the API response. e.g. "recordsWritten": "12420",

Reason: to avoid losing precision because JSON number types assume floating point.